composer.json

```
@@ -16,7 +16,7 @@
   },
   "require": {
     "php": ">=7.4",
-    "monolog/monolog": "~1.25"
+    "monolog/monolog": "^1.25"
```

Mmm I don't think that's what you wanted to achieve :confused: 
Your second version means that you allow from 1.25 to 2.0 (2.0 being excluded)
See https://getcomposer.org/doc/articles/versions.md#caret-version-range-
Your change should be more like this : 
```suggestion
    "monolog/monolog": "~1.25|~2.0"
```